src/exercise/dist/Main.js

```diff
@@ -0,0 +1,183 @@
+
```

Comment:
There are a few places with redundant empty lines - delete empty lines at the beginning of files and multiple consecutive empty lines